### ISSUE TITLE
Possible race condition? java.lang.NullPointerException: Cannot invoke "net.creeperhost.ftbbackups.org.quartz.Scheduler.clear()" because "net.creeperhost.ftbbackups.FTBBackups.scheduler" is null

### ISSUE THREAD

**Issue by mjevans** [EnigmaticaModpacks/Enigmatica8#214]
### Modpack Version

E8 2.1.0

### Describe your issue.

During a routine RCON scripted save-on stop the Enigmatica 8 server I run froze during shutdown.  The Forge console seemed partly responsive, but it was already mid stop and I didn't want to risk the world data.

A user disconnected / was still logged in during the saving players stage.  I imagine it's possible that caused a race condition in the user no longer existing / having been cleaned up by other threads.

Since I'm not using it anyway I've now removed the ftbbackups mod from the server.

### Latest Log

https://gist.github.com/mjevans/91a173abb00c477e08c66cb3d8c939ac

### Have you modified the modpack?

Yes

### User Modifications

Cherry picked the single type ore generation and akashic tome blacklist from development.
Disabled FTB backups (set enabled to false).

### Did the issue happen in singleplayer or on a server?

Server

**Comment by Raidobw2** [comment]
Hi, just jumping between trackers to share some info :)

Afaik there was a bug back then in which the backup thread locked itself, which caused an infinite freeze, but it was apparently fixed.

I'm aware of a similar issue which happens at random when closing a world (SP or MP). It happens on close since there's code that prevents closing the game if a backup is running, until it finishes. This causes the game to wait a maximum of 90 seconds (the amount of time before the fail safe kicks in and shuts down FTB Backups 2 forcefully). I know when that happens it looks bugged and it's always a 90 seconds timer.

I'll poke the devs again (the mod is actually made by CreeperHost, for FTB) about it probably Monday. I can link some of the info back here

TLDR: If game appears frozen wait a maximum of 90 seconds

Note about the current "error message": I think that's the backup thread being killed actually (intentionally by FTB Backups 2), but something else looks bugged since of the 90 seconds it takes for it to do anything. I get that the message isn't intuitive since it looks like an error, so I'll suggest them to improve it if at all possible

**Comment by mjevans** [comment]
It was 'hours later' not 90 seconds or the world timeout value.  Encountered when I first woke up after at least 5 hours of downtime after a middle of the night scheduled reboot triggered by rcon, didn't actually reboot.

**Comment by Raidobw2** [comment]
Interesting, I'll pass on the info, thanks!

**Comment by Raidobw2** [comment]
FYI apparently this problem was fixed in later builds of FTB Backups 2, possibly E8 needs to update the mod to fix this
https://github.com/FTBTeam/FTB-Mods-Issues/issues/314#issuecomment-1221495472

Reported version was 1.0.11, since then 2 more builds have been released, with the current version being 1.0.13 for Minecraft 1.18.2

**Comment by NielsPilgaard** [comment]
Ah nice, thanks!
